app/src/main/java/com/keylesspalace/tusky/StatusListActivity.kt

```diff
@@ -262,6 +276,24 @@ class StatusListActivity : BottomSheetActivity(), HasAndroidInjector {
                     }
                 }
             )
+
+            if (filterCreateSuccess) {
+                updateTagMuteState(true)
+                Snackbar.make(binding.root, getString(R.string.muting_hashtag_success_format, tag), Snackbar.LENGTH_LONG).apply {
+                    setAction(R.string.action_view_filter) {
+                        val intent = if (mutedFilter != null) {
+                            Intent(this@StatusListActivity, EditFilterActivity::class.java).apply {
+                                putExtra(EditFilterActivity.FILTER_TO_EDIT, mutedFilter)
+                            }
+                        } else {
+                            Intent(this@StatusListActivity, FiltersActivity::class.java)
```

Comment:
So v2 filters get EditFilterActivity, and v1 filters get FiltersActivity, is that right?

Reply:
At the moment, yes.

(As it not checks `mutedFilterV1`.)